src/cyw43_ll.c

```diff
@@ -466,6 +471,8 @@ static int cyw43_download_resource(cyw43_int_t *self, uint32_t addr, size_t raw_
     #endif
 
     for (size_t offset = 0; offset < len; offset += block_size) {
+        CYW43_EVENT_POLL_HOOK
```

Comment:
Please add a semicolon at the end of this line, to make it format like a statement.

Reply:
Done